src/app/components/text-image-component/text-image-component.component.ts

```diff
@@ -0,0 +1,19 @@
+import {Component, Input, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'app-text-image-component',
+  templateUrl: './text-image-component.component.html',
+  styleUrls: ['./text-image-component.component.css']
+})
+export class TextImageComponentComponent implements OnInit {
```

Comment:
Move the component from /src/app/components to /src/app/pages/homepage
Rename the component to just text-image so that it dosn't name it TextImageComponentComponent and just TextImageComponent. (if you rename the folder by right clicking on it and then refactor is should rename all the files)
Don't forget to declare the component in home-page.module.ts